### DISCUSSION TITLE
Can I use same Key in browser and mobile (Flutter SDK)?

### DISCUSSION THREAD

**Discussion by MizukiSonoko** [Web3Auth/Web3Auth#955] [Help]
Hello, In now I developing a web3 service using Web3Auth. it consists browser page / mobile app. 

# Architecture
- Browser (Vuejs / Nuxtjs / Single Page Application)
```
    "@web3auth/base": "^1.1.0",
    "@web3auth/coinbase-adapter": "^2.1.2",
    "@web3auth/ethereum-provider": "^1.1.0",
    "@web3auth/metamask-adapter": "^2.1.2",
    "@web3auth/openlogin-adapter": "^2.0.2",
    "@web3auth/web3auth": "^2.0.2",
```
- Mobile App (Flutter)
```
dependencies:
  web3auth_flutter: ^1.0.3
```

- Common Plug and Play
Same `Client ID` 

# Problem and Purpose
I have read  https://github.com/Web3Auth/web3auth-web/discussions/421 and
>  A unique key is generated for each combination of a verifier and clientId

But, I sign in with Google Login on mobile / browser, but I have received different key. 
How I should do?

**Comment by shahbaz17** [comment]
Hey @MizukiSonoko 

Both Web and Mobile should return the same key. Please update to the latest packages.

**Comment by MizukiSonoko** [comment]
@shahbaz17 
Thanks! It worked well in Flutter! I get same key in flutter!!
And I have faced new problem.... 
In iOS app using iOS SDK and Android app using Android SDK, even if I sign in with Google Login on mobile / browser, I have received different key.
How I should do?
